composeApp/src/main/kotlin/io/github/couchtracker/tmdb/TmdbPerson.kt

```diff
@@ -0,0 +1,10 @@
+package io.github.couchtracker.tmdb
+
+/**
+ * Class that represents a TMDB person.
+ * It holds no data, but provides the means to get all information,
+ * either by downloading them or loading them from cache.
+ */
+data class TmdbPerson(
```

Comment:
Should this also contain the language like tmdb movie has? I imagine people also have some sort of localized description.